Lib/defconQt/fontInfo.py

```diff
@@ -38,7 +32,7 @@ def __init__(self, font, parent=None):
         mainLayout.addWidget(buttonBox)
         self.setLayout(mainLayout)
 
-        self.setWindowTitle("Font Info – %s %s" % (
+        self.setWindowTitle(self.tr("Font Info") + " – %s %s" % (
             self.font.info.familyName, self.font.info.styleName))
```

Comment:
Okay. What about the ellipsiseseses? Replace by "\u2026"? Ugly for translators but would get around `pylupdate5`'s clumsiness.